### ISSUE TITLE
secure/insecure message witching does not work when sending pictures

### ISSUE THREAD

**Issue by donestefan** [signalapp/Signal-Android#8988]
- [x ] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
In normal text messages you can switch between sending secure/insecure transport by long pressing the "send" button. Long pressing stops working as soon as pictures are attached. 

Also, when pictures are attached, the message will be sent after a long press :(
Basically, while trying to switch to the correct transport, I unintentionally end up sending the message using the wrong transport :(

### Steps to reproduce
- open the android gallery
- share a picture with a signal contact
- long pressing the "send" button does not open the secure/insecure transport selection but instead sends the picture

### Device info
**Device:** Samsung SM-G930F (S7)
**Android version:** 8.0.0
**Signal version:** 4.45.2 (5182)

**Comment by greyson-signal** [comment]
There's a multitude of crazy and difficult to express scenarios you can get yourself into if you allow transport switching in the image send screen, most of them having to do with the fact that there's different attachment restrictions for MMS vs Signal messages. So we won't be allowing switching on that screen. However, I can see how it's frustrating if long-press sends there. Sounds reasonable to make long-press a no-op in that situation.

**Comment by UserX404** [comment]
I encountered the same issue but i figured out that after sending a regular message as sms in the add-picture-screen the send icon is toggled to unsecure.
It's not the expected behavior but it's a workaround.
